src/BlockTypes/CollectionPriceFilter.php

```diff
@@ -119,7 +117,7 @@ class="max"
 				<div
 					class="range"
 					style="<?php echo esc_attr( $range_style ); ?>"
-					data-wc-bind--style="state.filters.rangeStyle"
+					data-wc-bind--style="selectors.rangeStyle"
```

Comment:
I got this error:

![image](https://github.com/woocommerce/woocommerce-blocks/assets/5423135/c0d127da-7ccb-4ab4-bb28-804aa0fb6364)

Reply:
As this and the other two selector are placed under `state`, changing this line to state fix the error for me.
```suggestion
					data-wc-bind--style="state.rangeStyle"
```

Reply:
Not sure how I missed that, maybe I didn't push a change. thank you for spotting. 🙇🏻

Reply:
Looks like I somehow missed a couple other instances too. Fixed [here](https://github.com/woocommerce/woocommerce-blocks/pull/11943/commits/8e39030cd0f432d0ef3b11157f6b805b8d233d25)